<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #064663;
        color: #f2efe7;
        font-size: 14pt;
    }
    QLabel#logoLabel {
        color: #ffd345;
        font-weight: bold;
    }
    QLabel#descriptionLabel {
        border: 2px solid #ffd345;
        padding: 10px;
        background-color: #064663;
        color: #f2efe7;
    }
    QPushButton#nextButton {
        background-color: #041c32;
        color: #ffd345;
        font-weight: bold;
        padding: 10px;
        border-radius: 5px;
    }
    QPushButton#nextButton:hover {
        background-color: #0b5a7b;
    }
    QRadioButton {
        background-color: white;
        color: black;
        padding: 10px;
        border-radius: 4px;
    }
   </string>
  </property>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>640</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>PYLearn</string>
   </property>
  </widget>
  <widget class="QLabel" name="quizQuestion">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>471</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>150</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Which of the following best describes a Python set?
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="quizNumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>QUESTION 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="logoImage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap>pylogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>640</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>521</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Select the correct answer:</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radioButton_A_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>A. Ordered and immutable
</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_B_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>B. Unordered and contains duplicate element
</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_C_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>D. Unordered and contains only unique elements
</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_D_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>D. Software Engineer

</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
